span.go

```diff
@@ -105,7 +105,7 @@ func (s *spanImpl) Log(ld opentracing.LogData) {
 	defer s.onLog(ld)
 	s.Lock()
 	defer s.Unlock()
-	if s.trim() {
+	if s.trim() || s.tracer.options.DropAllLogs {
```

Comment:
I _think_ this is because the options were dynamic when trim() was originally introduced.  @tschottdorf to confirm.